app/assets/javascripts/hubstats/bootstrap.js

```diff
@@ -4,7 +4,7 @@
  * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
  */
 
```

Comment:
I added all of the semi colons because in Javascript, although it technically doesn't need semicolons everywhere, it can be helpful so that the compiler knows for sure where the developer wants to end its statements.... and Code Climate hated all of the missing semicolons. I went through line by line with Code Climate and added semicolons where they pointed out one was missing.